gradle/track-public-apis.gradle

```diff
@@ -82,4 +90,30 @@ android.libraryVariants.all { variant ->
             }
         }
     }
+
+    task checkApiKotlinValidator {
+        dependsOn apiCheck
+        group = 'Verification'
+        description = 'Update Kotlin binary compatibility validator generated public api files.'
+    }
+
+    task updateApiKotlinValidator {
+        dependsOn apiDump
+        group = 'Other'
+        description = 'Checks signatures of public API against the golden value in API folder generated by Kotlin binary compatibility validator.'
+    }
+
+    task checkApi {
+        dependsOn checkApiKotlinValidator
+        dependsOn checkApiMetalava
+        group = 'Verification'
+        description = 'Update generated public api files from both Metalava and Kotlin binary compatibility validator.'
+    }
+
+    task updateApi {
+        dependsOn updateApiKotlinValidator
+        dependsOn updateApiMetalava
+        group = 'Other'
+        description = 'Checks signatures of public API against the golden value in API folder generated by both Metalava and Kotlin binary compatibility validator.'
```

Comment:
Are the descriptions of checkApi and updateApi swapped somehow?

Reply:
swapped the dependsOn tasks 😄